File: web3rpc/rpc-specs/code-samples/python/klay/block/getBlockWithConsensusInfoByNumberRange.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = 1
numberRange = 10

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_block_with_consensus_info_by_number_range(blockHash, numberRange)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getBlockWithConsensusInfoByNumberRange.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockHashOrBlockNumber = 1
    const range = 10
    
    sdk.klay.getBlockWithConsensusInfoByNumberRange(blockHashOrBlockNumber, range, {}, (err, data, response) => {
        console.log(data);
    });
}
)()